### PR TITLE
Hipblaslt support

### PR DESCRIPTION
**Pull request by ramcherukuri**
hipBLASLt implementation in fused_dense

### CODE REVIEW

**Review by pruthvistony — CHANGES_REQUESTED** [review]
Please clean up the code.

**Review by pruthvistony — APPROVED** [review]
Initial implementation and code is tested.

File: setup.py [hunk 1]
```diff
@@ -351,6 +363,13 @@ def check_if_rocm_pytorch():
             )
         )
 
+#**********  fused dense  ****************
+        ext_modules.append(
```
Comment (lcskrishna) [inline setup.py:367]: Indentation is missing.

File: setup.py [hunk 2]
```diff
@@ -22,6 +22,15 @@
 this_dir = os.path.dirname(os.path.abspath(__file__))
 torch_dir = torch.__path__[0]
 
+
+def hipBLASlt_supported():
+    supported_arch = ['gfx942']
+    device_props = torch.cuda.get_device_properties(0);
```
Comment (jithunnair-amd) [inline setup.py:28]: This fails in situations where apex is being built as part of a Dockerfile, since GPUs are not visible at that stage.

**Pull request merged**